### PR TITLE
fix docker config dir

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

**Comment by KtorZ** [comment]
Ho, I can approve this :thinking:  ?

File: nix/docker.nix [hunk 1]
```diff
@@ -105,31 +105,44 @@ let
...
+            echo "Cleaning up"
         ''
         else "echo db-sync not supported on ${env.name} ; exit 1"}
     '')));
     entry-point = writeScriptBin "entry-point" ''
       #!${runtimeShell}
+      mkdir -p /config
+      if [ ! -f /config/pgpass ]
+      then
+        echo "Generating PGPASS file"
```
Comment (rhyslbw) [inline nix/docker.nix]: Logic looks good, but being a utility function for the service, I suggest hoisting this out into `/scripts/gen-pgpass-file.sh`, with the secrets directory passed in.

**Comment by craigem** [comment]
Rebased on master which has significantly improved the error messaging.

**Comment by craigem** [comment]
This is not ready to merge yet. I've fixed quite a few issues but the latest one is this error:

```
cardano-db-sync_1  | Errors in file: /tmp/migrate-2020-03-19T102151.log
```

For a variety of log files and then it exits.

I suspect it may be an issue with `cardano-db-sync` but I'm waiting for @erikd to confirm / deny.

File: nix/docker.nix (hunk 1 above)
Comment (rhyslbw) [reply nix/docker.nix]: Not required. Can always be refactored to later

**Comment by disassembler** [comment]
> This is not ready to merge yet. I've fixed quite a few issues but the latest one is this error:
> 
> ```
> cardano-db-sync_1  | Errors in file: /tmp/migrate-2020-03-19T102151.log
> ```
> 
> For a variety of log files and then it exits.
> 
> I suspect it may be an issue with `cardano-db-sync` but I'm waiting for @erikd to confirm / deny.

Look at your logs. You probably don't have postgres configured correctly. It works fine.

**Pull request merged**